sceptre/cli/list.py

```diff
@@ -153,3 +153,30 @@ def list_stacks(ctx, path):
     output = {f"{stack.name}.yaml": stack.external_name for stack in plan.graph}
     output_format = "json" if context.output_format == "json" else "yaml"
     write(output, output_format)
+
+
+@list_group.command(name="config")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def list_config(ctx, path):
```

Comment:
@jtfalkenstein comment from #1217 
> It doesn't really seem like a list command, though. Maybe render would be more accurate.

Reply:
@jfalkenstein It seems like a list command to me. We are listing the config keys. I guess it could be argued that we rendering too, although I would expect the output from render to be the jinja2 rendered template similar to `generate`. Thoughts?